### PR TITLE
chore(userspace/libsinsp): improve `get_event_type` util method in sinsp-example

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

> /area libsinsp

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: userspace/libsinsp/examples/util.cpp [hunk 1]
```diff
@@ -52,288 +52,9 @@ std::string get_event_category(ppm_event_category category)
...
-        case PPME_SOCKET_ACCEPT4_5_X: return "accept";
-        case PPME_SOCKET_SENDMMSG_E:
-        case PPME_SOCKET_SENDMMSG_X: return "sendmsg";
-        case PPME_SOCKET_RECVMSG_E:
-        case PPME_SOCKET_RECVMSG_X: return "recvmsg";
-        case PPME_SOCKET_RECVMMSG_E:
-        case PPME_SOCKET_RECVMMSG_X: return "recvmmsg";
-        default: return "UNKNOWN " + to_string(type);
-    };
+	if (type > 1 && type < PPM_EVENT_MAX)
```
Comment (Molter73) [inline userspace/libsinsp/examples/util.cpp:55]: Just out of curiosity, is this `type >= 1` meant to exclude the `PPME_GENERIC_E` and `PPME_GENERIC_X` types? Maybe we can change the condition to be a little more explicit about this?
```suggestion
	if (type < PPM_EVENT_MAX && type != PPME_GENERIC_E && type != PPME_GENERIC_X)
```

**Review by Molter73 — COMMENTED** [review]
Nice! Just a minor comment on making the check a little more explicit about what it is doing.

**Comment by FedeDP** [comment]
Great catch, thanks! @Molter73

**Review by Molter73 — APPROVED** [review]
/lgtm

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**